client/landing/jetpack-cloud/sections/backups/detail/index.jsx

```diff
@@ -11,27 +13,148 @@ import DocumentHead from 'components/data/document-head';
+import { emptyFilter } from 'state/activity-log/reducer';
+import { withLocalizedMoment } from 'components/localized-moment';
+import Gridicon from 'components/gridicon';
+import FoldableCard from 'components/foldable-card';
+import { Button } from '@automattic/components';
+import Filterbar from 'my-sites/activity/filterbar';
+import { updateFilter } from 'state/activity-log/actions';
+import getActivityLogFilter from 'state/selectors/get-activity-log-filter';
+import Pagination from 'components/pagination';
+// This can be used after merging https://github.com/Automattic/wp-calypso/pull/40066
+//import ActivityCard from '../../../components/activity-card';
+import { getEventsInDailyBackup } from '../utils';
+
+const PAGE_SIZE = 10;
 
 class BackupDetailPage extends Component {
+	changePage = pageNumber => {
+		this.props.selectPage( this.props.siteId, pageNumber );
+		window.scrollTo( 0, 0 );
+	};
+
 	render() {
-		const { siteId, backupId } = this.props;
+		const { backupId, filter, logs, moment, siteId, translate } = this.props;
+		const { page: requestedPage } = filter;
+
+		const backups = logs.filter( event => event.rewindId === backupId );
+		const thisBackup = backups[ 0 ];
+
+		const meta = thisBackup && thisBackup.activityDescription[ 2 ].children[ 0 ];
+		const metaList =
+			meta &&
+			meta.split( ', ' ).map( item => {
+				return <li key={ item }>{ item }</li>;
+			} );
+
+		const actualLogs =
+			( thisBackup && getEventsInDailyBackup( logs, new Date( thisBackup.activityDate ) ) ) || [];
+
+		const actualPage = Math.max(
+			1,
+			Math.min( requestedPage, Math.ceil( actualLogs.length / PAGE_SIZE ) )
+		);
+		const theseLogs = actualLogs.slice( ( actualPage - 1 ) * PAGE_SIZE, actualPage * PAGE_SIZE );
+
+		const cards = theseLogs.map( activity => (
+			<div key={ activity.activityId }>
+				<div>{ activity.activityDate }</div>
+				<div>{ activity.activityName }</div>
+			</div>
+			// This can be replaced with the following after we merge https://github.com/Automattic/wp-calypso/pull/40066
+			/*
+			<ActivityCard
+				{ ...{
+					key: activity.activityId,
+					moment,
+					activity,
+					allowRestore,
+				} }
+			/>
+			*/
+		) );
 
 		return (
 			<Main>
 				<DocumentHead title="Backup Details" />
 				<SidebarNavigation />
-				<div>Welcome to the backup detail page</div>
-				<div>Site ID: { siteId }</div>
-				<div>Backup ID: { backupId }</div>
+				<div>
+					<Gridicon icon="cloud-upload" />
+					{ thisBackup && moment( thisBackup.activityDate ).format( 'YYYY-MM-DD' ) }
+				</div>
+				<div>
+					<Button primary={ false }>{ translate( 'Download backup' ) }</Button>
+					<Button primary={ true }>{ translate( 'Restore to this point' ) }</Button>
+				</div>
+				<FoldableCard header={ translate( 'Total # of files backed up' ) }>
+					<ul>{ metaList }</ul>
+				</FoldableCard>
+				{ actualLogs.length > PAGE_SIZE ? (
+					<Fragment>
+						<div>{ translate( 'Backup details' ) }</div>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'View all backup details' )` *ES Score: 7*